invoices/migrations/0009_auto_20200502_0056.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.3 on 2020-05-01 21:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
please delete all migrations and regenerate a new one to have only one initial migration